scss/styles.scss

```diff
@@ -1,3 +1,15 @@
 * {
-    font-family: 'Roboto', sans-serif;
+  font-family: 'Roboto', sans-serif;
+}
+
+.body-container {
+  margin: 0 auto;
+  max-width: 1280px;
+  padding: 0 40px;
```

Comment:
Just change this to `padding: 100px` in order to center better the content and we are done.